src/modules/groupAdminMethods.ts

```diff
@@ -0,0 +1,27 @@
+import { TRPCError } from '@trpc/server';
+
+import { prisma } from '../utils/prisma';
+
+import { AddUserToGroupAdmin } from './groupAdminSchemas';
+import { tr } from './modules.i18n';
+
+export const groupAdminMethods = {
+    addUserToGroupAdmin: async (data: AddUserToGroupAdmin) => {
+        const groupAdmin = await prisma.groupAdmin.findUnique({
+            where: { userId_groupId: { userId: data.userId, groupId: data.groupId } },
+        });
+        if (groupAdmin) {
+            throw new TRPCError({ code: 'BAD_REQUEST', message: tr('User is already in group administrators') });
+        }
+
+        return prisma.groupAdmin.create({ data });
+    },
+
+    getGroupAdmins: async (groupId: string) => {
+        const groupAdmins = await prisma.groupAdmin.findMany({
+            where: { groupId },
+            include: { user: true, group: true },
```

Comment:
probably don't need to include group here, if you need to show a list of group admins you probably already fetched the group itself

Reply:
done